includes/wc-rest-functions.php

```diff
@@ -102,7 +102,8 @@ function wc_rest_upload_image_from_url( $image_url ) {
 	if ( ! $wp_filetype['type'] ) {
 		$headers = wp_remote_retrieve_headers( $response );
 		if ( isset( $headers['content-disposition'] ) && strstr( $headers['content-disposition'], 'filename=' ) ) {
-			$disposition = end( explode( 'filename=', $headers['content-disposition'] ) );
+			$content = explode( 'filename=', $headers['content-disposition'] );
```

Comment:
Missing align with the other variables.

Reply:
What do you mean with "missing align"?

Reply:
Change:

```php
			$content = explode( 'filename=', $headers['content-disposition'] );
			$disposition = end( $content );
			$disposition = sanitize_file_name( $disposition );
			$file_name   = $disposition;
```

to

```php
			$content     = explode( 'filename=', $headers['content-disposition'] );
			$disposition = end( $content );
			$disposition = sanitize_file_name( $disposition );
			$file_name   = $disposition;
```

Reply:
Thanks. I'll bear in mind for the future

Reply:
You can check for coding standards issues with PHPCS, already installed in this project, just install with `composer install` and then check with `composer run-script phpcs path-to-the-file.php` or fix some issues automatically with `composer run-script phpcbf path-to-the-file.php`